packages/core/lib/core.dart

```diff
@@ -1,6 +1,7 @@
 /// Core components for Widgetbook Demo App
 library core;
 
+export './src/dummy_data/dummy_data.dart';
```

Comment:
It's a bit weird that dummy data is exported from a package 🤔 I think we should take a different approach. What do you think?

Reply:
Moved to `app_widgetbook`